backend/tests/controllers/test_executions/test_test_executions.py

```diff
@@ -273,28 +273,25 @@ def test_report_test_execution_data(db_session: Session, test_client: TestClient
 def test_updates_test_execution(
     db_session: Session, test_client: TestClient, test_execution: TestExecution
 ):
+    new_review_decision = [
+        TestExecutionReviewDecision.APPROVED_FAULTY_HARDWARE.name,
+        TestExecutionReviewDecision.APPROVED_INCONSISTENT_TEST.name,
+    ]
     test_client.patch(
         f"/v1/test-executions/{test_execution.id}",
         json={
             "ci_link": "http://ci_link/",
             "c3_link": "http://c3_link/",
             "status": TestExecutionStatus.PASSED.name,
-            "review_decision": [
-                TestExecutionReviewDecision.APPROVED_FAULTY_HARDWARE.name,
-                TestExecutionReviewDecision.APPROVED_INCONSISTENT_TEST.name,
-            ],
+            "review_decision": list(new_review_decision),
             "review_comment": "Tests fail because of broken keyboard",
         },
     )
-
     db_session.refresh(test_execution)
     assert test_execution.ci_link == "http://ci_link/"
     assert test_execution.c3_link == "http://c3_link/"
     assert test_execution.status == TestExecutionStatus.PASSED
-    assert test_execution.review_decision == [
-        TestExecutionReviewDecision.APPROVED_FAULTY_HARDWARE.name,
-        TestExecutionReviewDecision.APPROVED_INCONSISTENT_TEST.name,
-    ]
+    assert set(test_execution.review_decision) == set(new_review_decision)
```

Comment:
hmm so runtime type of `test_execution.review_decision` is not a set? even though it's mapped as set?

Reply:
Yeah, I guess the set behaviour we get is because it is defined as set in `TestExecutionsPatchRequest`

Reply:
Sure, I now updated it to match this. `set` is not a native datatype in PostgreSQL, so the list data type on this level matches this as well. I also think keeping this data type on the request-level makes sense.

Reply:
It seems like we must keep this set data type on this level as well, otherwise mypy complains that we are trying to assign set to a list in the API logic. Another alternative would be to define everything as a list.

Reply:
I'd go with either everything is a list or we convert the request set to a list when passing to model

Reply:
set on model is currently sort of a lie

Reply:
True, I will keep the set on request-level but convert to list whenever assigning to the DB model